context/MqlContext/MqlContext.ts

```diff
@@ -0,0 +1,60 @@
+import { Client } from "urql";
+import { createContext } from "react";
+
+export type MqlContextType = {
+  /*
+    The MQL Server URL can be overridden by a user in order to access a development
+    server or a test server as needed. This value represents the currently selected 
+    URL for the user and is not optional.
+  */
+  mqlServerUrl?: string | null;
```

Comment:
Is not optional? Do we want the `?` then?  Or would we prefer a type error to throw?

Reply:
This is optional in the sense that on initial load it is not yet retrieved from the server.